site/src/components/card.md

```diff
@@ -0,0 +1,203 @@
+
+```html
+<div class="card width-300">
+	<div class="card-template">
+		<p class="card-supertitle">Course</p>
+		<a href="#" class="card-title">Learn how to create a card component</a>
+		<img
+			aria-hidden="true"
+			class="card-template-icon"
+			src="https://docs.microsoft.com/en-us/learn/achievements/review-microsoft-azure-pricing-slas-lifecycles.svg"
+		/>
+		<div class="card-template-detail">
+			<p>Additonal details or elements can be placed here.</p>
+		</div>
+	</div>
+	<div class="card-footer">
+		<div class="card-footer-item">
+			<div class="buttons">
+				<button class="button button-clear button-sm">Save</button>
+				<button class="button button-primary button-filled button-sm">Start</button>
+			</div>
+		</div>
+	</div>
+</div>
+```
+
+## Patterns for more accessible content
+
+In each card, you'll also be presented with a choice for what element to assign to the card's title text. It can be tempting to assign this a heading value outside or inside an anchor (`h1-h6 > a`), but be wary that when many cards are presented together, this pattern can severely muddy landmark navigation for screenreader users. For this reason, if a section presents a large set of cards, especially those that lack significant contextual information beyond the title, the recommended element is a simple anchor (`<a>`) for a linked card.
+
+As you'll see in the examples below, the recommended containing element to pair with a `.card` is [`<article>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article).
+
+Additionally, you can increase the the clickable area of a card with the `.stretched-link` class. However, you should ensure this only happens on cards with a single clickable element. Visit the [stretched link documentation for example usage](./stretched-link.md).
+
+## Card's main subcomponents
+
+The card element encapsulates several subcomponents.
+
+- `.card-header` - an optional section on top of the card.
+- `.card-template` - a highly prescriptive layout for the card's main contents. Not intented for compatibilty with `.card-content`.
+- `.card-content` - a free-form box within which a card's main content can be placed. Intended for use instead of `.card-template` not in addition to it.
+- `.card-footer` - an optional section at the bottom of the card. Usually contains actions relevant to the card's subject.
+
+Each of these subcomponents has their own optional subcomponents. They are discussed below.
+
+## The main content area of card
+
+In styling the main content area of a card, it's recommended you look first to `.card-template`, and if that fails, fall back to the less prescriptive `.card-content`.
+
+Both `.card-template` and `.card-content` support the use of several subcomponents. Because they can be used in both `.card-template` and `.card-content`, they are no namespaces by their parent elements, but rather only by `.card-`
```

Comment:
"they are no namespaces ..." reads odd. Should it be "they are not namespaced ..."? not sure if namespace can be a verb.